modules/file/api_test.go

```diff
@@ -0,0 +1,108 @@
+package file_test
+
+import (
+	"errors"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/kroppt/voxels/modules/file"
+)
+
+func stringSliceEqual(a, b []string) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i := range a {
+		if a[i] != b[i] {
+			return false
+		}
+	}
+	return true
+}
+
+func TestModuleNew(t *testing.T) {
+	t.Run("return is non-nil", func(t *testing.T) {
+		mod := file.New()
+		if mod == nil {
+			t.Fatal("expected non-nil return")
+		}
+	})
+}
+
+func TestModuleGetReadCloser(t *testing.T) {
+
+	t.Run("open empty file", func(t *testing.T) {
+		expectErr := io.EOF
+		expectN := 0
+		mod := file.New()
+
+		readCloser, err := mod.GetReadCloser("empty.txt")
```

Comment:
I think naming this `test-empty.txt` would better clarify and make more consistent.